packages/slice-machine/src/theme.ts

```diff
@@ -149,19 +150,19 @@ const AppTheme = (): Theme =>
         margin: 0,
         pl: 2,
       },
-      pre: {
-        variant: "prism",
-        fontFamily: "monospace",
-        fontSize: 1,
-        my: 1,
-        color: "textClear",
-        bg: "grey01",
-        p: "2px",
-        overflow: "auto",
-        code: {
-          color: "inherit",
-        },
-      },
+      // pre: {
```

Comment:
if it's not use anymore please remove it :)

Reply:
will do :)